openmeter/billing/adapter/profile.go

```diff
@@ -0,0 +1,319 @@
+package billingadapter
```

Comment:
I thought we handle DB operation in `repository`

Reply:
https://github.com/openmeterio/openmeter/pull/1544#discussion_r1768339413